packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -37,10 +37,10 @@ class Variables extends StatelessWidget {
     );
   }
 
-  void onItemPressed(DartObjectNode v, DebuggerController controller) {
+  void onItemPressed(DartObjectNode v, DebuggerController controller) async {
```

Comment:
Can this method return `Future<void>`? This way we can get warnings if the caller does not await on it (btw there is a warning for this as well: https://dart-lang.github.io/linter/lints/avoid_void_async.html)